cmd/solution/init.go

```diff
@@ -109,12 +109,11 @@ func generateSolutionPackage(cmd *cobra.Command, args []string) {
 	createSolutionManifestFile(solutionName, manifest)
 }
 
-func createInitialSolutionManifest(solutionName string, solutionType string) *Manifest {
+func createInitialSolutionManifest(solutionName string, solutionType string, solutionVersion string) *Manifest {
```

Comment:
why do we want the solution version required?
If we want to make this extensible, I will propose using Golang idiom with things like WithSolutionVersion(), WithManifestVersion(), etc.